dependencies.props

```diff
@@ -9,6 +9,7 @@
   <!-- Auto-upgraded properties for each build info dependency. -->
   <PropertyGroup>
     <CoreFxExpectedPrerelease>beta-24911-02</CoreFxExpectedPrerelease>
```

Comment:
The config would need to be added here and an entry added to dotnet/versions subscriptions.json, but I don't think there would be anything unusual about it vs. other subscriptions. I think it would be a good idea to auto-update if this ends up with the new dependency.